easybuild/easyconfigs/o/OpenMPI/OpenMPI-2.0.1-GCC-6.2.0-2.27.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenMPI'
+version = '2.0.1'
+
+homepage = 'http://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-2 implementation."""
+
+toolchain = {'name': 'GCC', 'version': '6.2.0-2.27'}
+
+sources = [SOURCELOWER_TAR_GZ]
+
+source_urls = ['http://www.open-mpi.org/software/ompi/v%(version_major_minor)s/downloads',]
+
+dependencies = [('hwloc', '1.11.4')]
+
+configopts = '--with-threads=posix --enable-shared --enable-mpi-thread-multiple --with-verbs '
+configopts += '--enable-mpirun-prefix-by-default '  # suppress failure modes in relation to mpirun path
+configopts += '--with-hwloc=$EBROOTHWLOC '  # hwloc support
+configopts += '--disable-dlopen '  # statically link component, don't do dynamic loading
+configopts += '--with-tm=/usr/local/packages/adaptive/torque ' # we want torque support as well
+
+# Add in support for hardware from Mellanox OFED
+configopts += '--with-mxm=/opt/mellanox/mxm '
+configopts += '--with-fca=/opt/mellanox/fca '
+#configopts += '--with-hcoll=/opt/mellanox/hcoll '
+configopts += '--with-knem=/opt/knem-1.1.2.90mlnx1 ' # knem comes from Mellanox OFED
+configopts += '--with-platform=contrib/platform/mellanox/optimized '
```

Comment:
Yes it is, I thought I had removed all of those lines before pushing but I must have copied the wrong file across. It's changed now.